src/components/Daos/DaosPage.tsx

```diff
@@ -154,69 +182,65 @@ class DaosPage extends React.Component<IProps, IState> {
                 </p>
               }
             >
-              {daoNodes}
+              {otherDaoNodes}
             </InfiniteScroll> : "None"}
         </div>
       </div>
     );
   }
 }
 
+const createSubscriptionObservable = (props: IStateProps, data: SubscriptionData = null) => {
+  const arc = getArc();
+  const { currentAccountAddress, followingDAOs } = props;
+
+  // TODO: right now we don't handle a user following or being a member of more than 100 DAOs
+  //       it was too hard to figure out the UI with infinite scrolling in this case we would need a different UI
```

Comment:
Is infinite scroll more feasible after the change that has only one search box?

Reply:
no

Reply:
cant really do two lists with infinite scroll on both on the same page easily... unless they are in separate columns or tabs